libs/libavro/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# libavro - Makefile for Apache AvroC library
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=avro-c
```

Comment:
I think this sane for now, if one wants to add more bindings it should be converted to into a "meta" with build variants.